importer/importer.py

```diff
@@ -196,6 +196,7 @@ def get_items(connection,
         print("USING RANDOM SAMPLE OF " + str(short))
     filename = specific_table_name + "_" + utils.get_current_timestamp()
     problem_reports = []
+    wikidata_site = utils.create_site_instance("wikidata", "wikidata")
     for row in database_rows:
         monument = class_to_use(row, mapping, data_files, existing)
```

Comment:
Add repo as an argument in Monument.__init__, and pass it here. That way you can use it for calls to e.g. `get_P31`.